modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-dropdown/ddm-template.json

```
@@ -0,0 +1,7 @@
+{
+	"className": "com.liferay.template.model.TemplateEntry",
+	"ddmTemplateKey": "RELEASE NOTES DROPDOWN",
```

It's better to add slash for keys.

```suggestion
	"ddmTemplateKey": "RELEASE-NOTES-DROPDOWN",
```